### PR TITLE
fix: make button not disappearing

### PR DESCRIPTION
**Pull request by WaDadidou**

### CODE REVIEW

**Review by FerrymanGMI — APPROVED** [review]
lgtm

**Review by zxxma — DISMISSED** [review]
LGTM 👍

File: packages/components/TopMenu/ConnectWalletButton.tsx [hunk 1]
```diff
@@ -26,8 +28,8 @@ export const ConnectWalletButton = () => {
   return (
     <>
       <View style={{ marginRight: headerMarginHorizontal }}>
-        {areThereWallets ? (
-          <TopMenu />
+        {areThereWallets && !!selectedWallet ? (
+          <TopMenu selectedWallet={selectedWallet} />
```
Comment (hthieu1110) [inline packages/components/TopMenu/ConnectWalletButton.tsx:32]: is that enough if we just use ```
{!!selectedWallet ? (
          <TopMenu selectedWallet={selectedWallet} />
```

Comment (WaDadidou) [reply packages/components/TopMenu/ConnectWalletButton.tsx]: You're right, I removed

**Review by omniwired — APPROVED** [review]

**Review by zxxma — APPROVED** [review]
LGTM 👍

**Pull request merged**